proteus/mprans/BoundaryConditions.py

```diff
@@ -311,18 +411,42 @@ def __cpp_UnsteadyTwoPhaseVelocityInlet_vof_dirichlet(self, x, t):
         xx[2] = x[2]
         return self.waves.__cpp_calculate_vof(xx, t)
 
-    # FOLLOWING BOUNDARY CONDITION IS UNTESTED #
-    def setTwoPhaseVelocityInlet(self, U, waterLevel, vert_axis=None, air=1.,
-                                 water=0.):
+
+    def setTwoPhaseVelocityInlet(self, U, waterLevel, smoothing, Uwind=None,
+                                 vert_axis=None, air=1., water=0.,
+                                 kInflow=None, dissipationInflow=None,
+                                 kInflowAir=None, dissipationInflowAir=None):
```

Comment:
@adimako Yes, I used it for different test problems (either "1phase" and 2phase cases)